### ISSUE TITLE
windows cmake 失败

### ISSUE THREAD

**Issue by matr1xes** [li-plus/chatglm.cpp#20]
E:\github_code\chatglm.cpp\chatglm.cpp(14,10): fatal  error C1083: 无法打开包括文件: “sys/mman.h”: No such file or directory [E:\github_code\chatglm
.cpp\build\chatglm.vcxproj]

**Comment by foldl** [comment]
FYI: PR #24.

**Comment by iHaagcom** [comment]
Same issue here trying to compile no cmake project files can be generated

**Comment by foldl** [comment]
@iHaagcom are you testing the latest code with PR #24 merged?

**Comment by iHaagcom** [comment]
Yes, I’ll try again.

**Comment by iHaagcom** [comment]
Yes it’s unable to compile the ChatGLM.cpp all the other exes for sentence piece and spm_train compile but errors with chatglm.cpp
